src/js/indexes/components/File.js

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import styled from "styled-components";
+import { getFontWeight } from "../../app/theme";
+import { BoxGroupSection } from "../../base";
+import { byteSize } from "../../utils/utils";
+
+const StyledFile = styled(BoxGroupSection)`
+    display: flex;
+`;
+
+const StyledSize = styled.div`
+    margin-left: auto;
+    font-weight: ${getFontWeight("bold")};
+`;
+
+export function File({ fileName, download_url, id, size }) {
```

Comment:
Looks like there is no universal preference for declarations vs arrow functions. However, let's be consistent within Virtool. Almost everything is an arrow function right now, so let's stick with that.